plugin/trino-hive/src/main/java/io/trino/plugin/hive/orc/OrcDeletedRows.java

```diff
@@ -217,8 +236,13 @@ private RowId getRowId(int position)
                         if (page != null) {
                             for (int i = 0; i < page.getPositionCount(); i++) {
                                 long originalTransaction = BIGINT.getLong(page.getBlock(ORIGINAL_TRANSACTION_INDEX), i);
+                                int bucketData = toIntExact(INTEGER.getLong(page.getBlock(BUCKET_ID_INDEX), i));
```

Comment:
"bucketValue"?

Reply:
Renamed to `encodedBucketValue` here an above